.github/workflows/cr.yml

```diff
@@ -0,0 +1,23 @@
+name: Continuous Releases
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - uses: pnpm/action-setup@v4
+        with:
+          run_install: true
+
+      - name: Compile
+        run: pnpm build
+
+      - name: Release
+        run: pnpm dlx pkg-pr-new publish
```

Comment:
we have a workflow for this already, it's better that we use it.

Reply:
I found this: https://github.com/beyonk-group/workflows/blob/main/.github/workflows/canary-libs.yml

Any examples I can check on how to integrate this? Or is it copy-pasta? 😁

Reply:
https://github.com/beyonk-group/lander/blob/master/.github/workflows/canary-libs.yml

Reply:
Started using this, now the Action doesn't run 🤔

Reply:
Ah IG its because workflows is a private repo while this one is public